cpp/config/suites/cpp/metrics

```diff
@@ -35,7 +35,7 @@
 + odasa-cpp-metrics/Classes/CInheritanceDepth.ql: /Metrics/Encapsulation
 + odasa-cpp-metrics/Classes/CLackOfCohesionCK.ql: /Metrics/Encapsulation
 + odasa-cpp-metrics/Classes/CSizeOfAPI.ql: /Metrics/Encapsulation
-+ odasa-cpp-metrics/Classes/CSpecialisation.ql: /Metrics/Encapsulation
++ odasa-cpp-metrics/Classes/CSpecialization.ql: /Metrics/Encapsulation
```

Comment:
I'm not quite sure what `odasa-cpp-metrics` is (there are 10 files that mention it, but there are no actual file paths matching it)...

Anyway, this is/was `specialisation`

Reply:
I've dropped this.